Graphs/Table/index.js

```diff
@@ -77,9 +77,9 @@ class Table extends AbstractGraph {
                 event: events.REMOVED_COLUMNS
             });
         }
-        
-        if(!objectPath.has(scrollData, [`selectedRow_${requestId}`]) && !isEmpty(this.selectedRows)) {
-            this.updateTableStatus({ [`selectedRow_${requestId}`]: this.selectedRows })
+        const rowsInStore = objectPath.has(scrollData, 'selectedRow') ? objectPath.get(scrollData, 'selectedRow') : {}
```

Comment:
You should add this logic in updateTableStatus since for every call of updateTableStatus you seem to get the rows from the store and add to it.